lib/src/lib/table/models/table-cell.model.ts

```diff
@@ -81,4 +81,13 @@ export interface TableCell<T = any> {
    */
   footer?: ((data: T[], cell: TableCell) => string) | string;
   footerCellClass?: string;
+
+  /**
+   * Cell prefixed icon, use cellFormatter to remove text if you want to only use the icon
```

Comment:
There's a default text? is this the `-`, why not removing automatically if a icon exists?

Reply:
I didn't want to do that because you might want to have a text and an icon in your table cell. I do have some use cases for that where I do it via css now.

Reply:
There is no - default, if there's no string value in your key it will just be an empty cell. See:     <ng-container *ngIf="cellValue">